coverage_comment/github.py

```diff
@@ -153,6 +153,8 @@ def post_comment(
             raise CannotPostComment from exc
 
 
-def set_output(**kwargs: bool) -> None:
-    for key, value in kwargs.items():
-        print(f"::set-output name={key}::{json.dumps(value)}")
+def set_output(config: settings.Config, **kwargs: bool) -> None:
+    if config.GITHUB_OUTPUT:
+        with open(config.GITHUB_OUTPUT, "a") as f:
+            for key, value in kwargs.items():
+                f.write(f"{key}={json.dumps(value)}\n")
```

Comment:
rather than passing the whole config, maybe we could just pass `config.GITHUB_OUTPUT` so that the function would be simpler. (and easier to test)